BDArmory/UI/BDArmorySetup.cs

```diff
@@ -3342,6 +3342,23 @@ void WindowSettings(int windowID)
 
                 if (BDArmorySettings.ADVANDED_USER_SETTINGS)
                 {
+                    BDArmorySettings.MISSILE_CM_SETTING_TOGGLE = GUI.Toggle(SLineRect(++line), BDArmorySettings.MISSILE_CM_SETTING_TOGGLE, Localizer.Format("#LOC_BDArmory_Settings_MissileCMToggle"));
```

Comment:
These might be better suited in one of the other sections instead of "Other Settings", e.g., "Gameplay Settings".